src/cmake/AxomConfig.cmake

```diff
@@ -158,8 +158,8 @@ install(
 # Add it to a temporary list before creating the cache variable to use list(APPEND)
 set(_axom_exported_targets axom)
 
-blt_list_append(TO _axom_exported_targets ELEMENTS cuda cuda_runtime IF ENABLE_CUDA)
-blt_list_append(TO _axom_exported_targets ELEMENTS hip hip_runtime IF ENABLE_HIP)
+blt_list_append(TO _axom_exported_targets ELEMENTS cuda cuda_runtime IF AXOM_ENABLE_CUDA)
+blt_list_append(TO _axom_exported_targets ELEMENTS hip hip_runtime IF AXOM_ENABLE_HIP)
```

Comment:
We should use the macro (which is basically copy/pasted here) but we do a couple things that the macro should do, which is to add extra targets to export. Because of this we would have to either enhance the macro or duplicate the same code.

Since I want to move away from exporting BLT's targets, I figure we would leave it until the new way is ready.